### PR TITLE
Request to merge Deploy

### PR DESCRIPTION
**Pull request by Kingstalux**
- <a href="https://www.loom.com/share/4dc304140b27489c978c2585152a0154">Loom Presentation</a> of the project.
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API is stored in the Redux store and then filtered using a Filter stateless component.
- Every page (the home page and the details for each country) has a unique route within the SPA.
- The project is deployed using Heroku and is accessible online.
- Tests have been carried out using react testing library and jest.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,16 @@
-import logo from './logo.svg';
-import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import Details from './components/Details';
+import Home from './components/Home';
+import NavBar from './components/NavBar';
 
 function App() {
```
Comment (rloterh) [inline src/App.js:6]: - [x] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

Comment (rloterh) [inline src/App.js:6]: The styling of the list on your Homepage does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), and therefore difficult to differentiate the details of each list item. Also, your design is overflowing/overlapping on mobile screens. Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that:
- [x] Your design **fits** perfectly on mobile screens
- [x] The cards/ cells on the _Home page_ are **two columns** on mobile.
- [x] The cells/ card of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.



**Your design:**
image

image

**The original design:**
image

File: README.md [hunk 2]
```diff
@@ -1,70 +1,91 @@
 # Getting Started with Create React App
```
Comment (rloterh) [inline README.md:2]: Your _ReadMe_ needs some improvements so as to make it appear more professional. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this capstone project,  which will likely feature on your portfolio. Therefore, kindly ensure to improve upon these aspects of your ReadMe:
- [x] _Project description_, _how to set up_, _run and test this project_ **should be detailed, clear, and well** organized.
- [x] The _code_ aspects in each procedure should be quoted with **backticks**.
- [x] Include necessary screenshots, project demo links where applicable so as to give a birds-eye view of your project
- [x] All _typos_ **should be corrected**. You can crosscheck by installing the [Grammarly plugin](https://www.grammarly.com/)

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Kingsely, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to complete this project but you are almost there! 🚦  🚦  

## Good Points 👍 
- App is correctly deployed
- Good video presentation of the project
- Redux is effectively used to manage state data
- Test cases are passing
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Kingstalux) [reply README.md:2]: The README file contains the following:
- A brief description of the project and how to set it up in the local environment and run the tests.
- A preview of the project in the form of a gif.
- A presentation of the project and a link to the live page deployed online.

File: src/App.js (hunk 1 above)
Comment (Kingstalux) [reply src/App.js:6]: Okay.

Comment (Kingstalux) [reply src/App.js:6]: The following changes have been applied to the design:
-  The mobile design fits perfectly on mobile screens
- The cards on the Home page are two columns on mobile.
- The cards follow the alternating dual-color pattern. Each card of the list item should now have surrounding cards having different (alternating) background colors.
- Scre

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**